### PR TITLE
Populate the type of record is created or updated

### PR DESCRIPTION
**Pull request by prashantmrshine**

### CODE REVIEW

**Review by am-shakeel — CHANGES_REQUESTED** [review]
Thanks for your submission. It appears that this pull request contains more files than described in the title. Please review and update the pull request to either remove the extra/unintended files or to update the description to be more comprehensive. Ensure your contributions are meaningful and adhere to the project’s standards. [If applicable: You can create a new branch on your forked repository to avoid accidental inclusions of new commits in the future.]. Closing this for now. Once you make additional changes, feel free to re-open this Pull Request or create a new one.

**Comment by am-shakeel** [comment]
Some files are already here - https://github.com/ServiceNowDevProgram/code-snippets/pull/1450. 
Please avoid duplication

**Comment by prashantmrshine** [comment]
Hi @am-shakeel , What is duplicated, which is the extra file?

File: GlideRecord/Display list of records based on Users Location/Readme.md
Comment (wiz0floyd) [inline GlideRecord/Display list of records based on Users Location/Readme.md:1]: This file is part of another PR. Please remove it from this PR.

File: GlideRecord/Display list of records based on Users Location/listOfRecordsBasedOnLocation.js
Comment (wiz0floyd) [inline GlideRecord/Display list of records based on Users Location/listOfRecordsBasedOnLocation.js:1]: This file is part of another PR

**Review by wiz0floyd — CHANGES_REQUESTED** [review]

File: GlideRecord/Display list of records based on Users Location/Readme.md
Comment (wiz0floyd) [reply GlideRecord/Display list of records based on Users Location/Readme.md:1]: In your branch make a commit that deletes the files. 

https://docs.github.com/en/repositories/working-with-files/managing-files/deleting-files-in-a-repository

**Pull request closed without merging**